### PR TITLE
Add issue templates per #25

### PR DESCRIPTION
**Pull request by andrewnicols**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Bug in Moodle LMS?
+    url: https://tracker.moodle.org/
+    about: Found a problem with Moodle LMS? Report it to the Moodl etracker
```
Comment (sarjona) [inline .github/ISSUE_TEMPLATE/config.yml:5]: There is a tiny typo: Moodl  etracker ;-)

File: .github/ISSUE_TEMPLATE/documentation-bug.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  attributes:
+    label: What is the issue with this page?
+  validations:
+    required: true
+- type: dropdown
+  id: canHelp
+  attributes:
+    label: Are you able to provide a patch for this?
+    options:
+      - No (Defeault)
```
Comment (sarjona) [inline .github/ISSUE_TEMPLATE/documentation-bug.md:29]: Tiny typo: Defeault

**Review by sarjona — CHANGES_REQUESTED** [review]

**Comment by andrewnicols** [comment]
Fixed those :)

**Review by sarjona — APPROVED** [review]
Thanks Andrew! These templates look great and I think they are a great idea! <3

**Pull request merged**